extractor/test_dandelion.py

```diff
@@ -0,0 +1,34 @@
+# SAMPLE CODE FOR DANDELION API
+# API token from @achaccha
+
+import requests
+import json
+ 
+TOKEN = '0af76af32cb343a19f7c7ea0e153922e'
```

Comment:
Is this API token? Please do not store this on a file (it's a password for you), but instead use environmental variable to set this value at code execution time. Here's one example package that allows you to do this:

https://github.com/theskumar/python-dotenv

Reply:
I already integrated `python-dotenv` to our project. So just make sure your `.env` exists.

Reply:
I'll make our `.env file. 
Thank you for advice.